Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/repository/VaccinationRepository.kt

```diff
@@ -22,4 +77,9 @@ class VaccinationRepository @Inject constructor() {
     fun clear() {
         throw NotImplementedError()
     }
+
+    fun deleteVaccinationCertificate(certificateId: String) =
```

Comment:
Should we make this `suspend`? Maybe the function above it too. UI Probably wants to block until the call is done?